apps/occurrences/manager.py

```diff
@@ -21,13 +22,16 @@ def search(
         suppress_hidden: bool = True,
     ) -> 'SearchableModelQuerySet':
         """Return search results from apps.occurrences."""
+
+        if not query:
+            es_qs = super().search(suppress_unverified=suppress_unverified, suppress_hidden=suppress_hidden)
```

Comment:
If the queryset doesn't necessarily comes from ElasticSearch, can we rename `es_qs` to just `qs`?